composer.json

```diff
@@ -15,11 +15,11 @@
         "php": ">=5.3.0"
     },
     "autoload": {
-        "psr-0": { "Pimple": "src/" }
+        "psr-4": { "Pimple//": "src/Pimple/" }
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "3.0.x-dev"
+            "dev-master": "3.0-dev"
```

Comment:
but changing from the proper version to the version supported by Composer only to avoid common mistakes looks weird to me. The actual alias version will be ``3.0.x-dev``